python/lsst/dax/imgserv/fitsMetadataSchema.py

```diff
@@ -0,0 +1,59 @@
+
+
```

Comment:
I'm really tired of the copyrights (and I'm definitely not reviewing them). I remember someone mentioning (at Bremerton?) that there should be enough to have one `COPYRIGHT` file at the top level, or was I dreaming?